spring-cloud-app-broker-acceptance-tests/src/test/java/org/springframework/cloud/appbroker/acceptance/CloudFoundryAcceptanceTest.java

```diff
@@ -451,11 +477,87 @@ private WebClient getSslIgnoringWebClient() {
 
 	protected Mono<List<ApplicationDetail>> getApplications(String app1, String app2) {
 		return Flux.merge(cloudFoundryService.getApplication(app1),
-			cloudFoundryService.getApplication(app2))
+				cloudFoundryService.getApplication(app2))
 			.parallel()
 			.runOn(Schedulers.parallel())
 			.sequential()
 			.collectList();
 	}
 
+	private void prepareCLI() {
+		try {
+			cfHome = Files.createTempDirectory("app-broker-acceptance-tests").toString();
+
+			callCLICommand(List.of("cf", "login", "-a",
+				cloudFoundryProperties.getApiHost(),
+				"--skip-ssl-validation", "-u",
+				cloudFoundryProperties.getUsername(),
+				"-p",
+				cloudFoundryProperties.getPassword(),
+				"-o", "test-instances"))
+				.block(Duration.ofSeconds(60));
+			callCLICommand(List.of("cf", "install-plugin", "-f", "-r", "Cf-Community", "Service Instance Logging"))
+				.block(Duration.ofSeconds(60));
+		}
+		catch (IOException e) {
+			throw new RuntimeException(e);
+		}
+	}
+
+	protected Mono<String> callCLICommand(List<String> command) {
+		return Mono.fromCallable(() -> {
+			if (LOG.isDebugEnabled()) {
```

Comment:
Did you mean the `if (LOG.isDebugEnabled()) {` ? 
We do have a PMD rule to always guard log calls https://github.com/spring-cloud/spring-cloud-app-broker/blob/e7bd5d6981867895470c4fd8fece4d1dc39a2ba0/src/pmd/pmdRuleSet.xml#L23-L25
I agree that this is not saving much in test code but in production code it's a very common practice to avoid processing resource-consuming `toString` calls.

Reply:
Should I remove these guards in test code @spikymonkey or leave them for consistency?

Reply:
Ah I didn't realise we had a PMD rule. Yeah I understand the common practice. I feels like overkill to me in tests (and in a lot of cases in production code too, unless we're talking about a hot path, but I realise that's probably a controversial take 😅) but I don't mind the consistency.